Kiosk/Models/InkRecognizerExplorer/InkRecognitionUnit.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Collections.Generic;
+
+namespace IntelligentKioskSample.Models.InkRecognizerExplorer
+{
+    public class InkRecognitionUnit
+    {
+        public List<Alternate> alternates { get; set; }
+        public BoundingRectangle boundingRectangle { get; set; }
+        public string category { get; set; }
+        public string @class { get; set; }
```

Comment:
>@class [](start = 22, length = 6)

iv'e never seen this before what does it do?

Reply:
The word "class" is a reserved word in C# so placing the @ symbol before it allows it to be used as a property like normal.

Reply:
Removing the property since it is not being used.